src/Workspaces/Core/Portable/Shared/Utilities/AsyncBatchingDelay.cs

```diff
@@ -1,50 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Shared.TestHooks;
-using Roslyn.Utilities;
-
-namespace Roslyn.Utilities
-{
-    internal sealed class AsyncBatchingDelay
-    {
-        private readonly AsyncBatchingWorkQueue<bool> _workQueue;
```

Comment:
this already was just a no generic arg wrapper around AsyncBatchignWorkQueue.  So just unified on that type instead.